pkg/auth/auth.go

```diff
@@ -87,7 +87,7 @@ func tokenForHost(cfg *config.Config, host string) (string, string) {
 }
 
 func tokenFromGh(path string, host string) (string, string) {
-	cmd := exec.Command(path, "auth", "token", "--hostname", host)
+	cmd := exec.Command(path, "auth", "token", "--secure-storage", "--hostname", host)
```

Comment:
@samcoe Could there be downsides on a go-gh release relying on an yet-unreleased gh feature?

Reply:
I don't see any issue as the errors are being properly handled. I tested it on one of my extensions and it worked out fine.